### PR TITLE
Add "digest" command

### LINKED ISSUE #477: digest command
Primarily to facilitate user's access to `dandi-etag` digest which is to be introduced in https://github.com/dandi/dandi-cli/pull/474 .  Could be folded into #474 or developed independently after/on top.  Should by default provide our `dandi-etag` but allow for estimation of other digests we have supported in Digester as well, in particular md5 and sha256.

### PR DESCRIPTION
**Pull request by jwodder**
Closes #477.

### CODE REVIEW

**Comment by jwodder** [comment]
The test failures are due to #479 not being included in this PR.

File: dandi/cli/cmd_digest.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    show_default=True,
+)
+@click.argument("paths", nargs=-1, type=click.Path(exists=True, dir_okay=False))
+@map_to_click_exceptions
+def digest(paths, digest_alg):
+    """ Calculate file digests """
+    from ..support.digests import get_digest
+
+    for p in paths:
+        print(f"{p}:", get_digest(p, digest=digest_alg))
```
Comment (yarikoptic) [inline dandi/cli/cmd_digest.py:25]: please add a rudimentary unit test similar to the ones we have for other `cmd_` code

Comment (jwodder) [reply dandi/cli/cmd_digest.py:25]: Done.

**Comment by yarikoptic** [comment]
Thank you! (I will assume that all CI fails are due to upload RFing...)

**Pull request merged**